### DISCUSSION TITLE
SidePanel should not scroll with page + bonus

### DISCUSSION THREAD

**Discussion by brucegomes** [IntelliTect/EssentialCSharp.Web#13] [Feedback and Ideas] [answered]
- On Really long pages, it takes lots and lots of scrolling back up to get to the side panel content so I can go to the next thing I want. It would be nice If when open, it would always stay visible at the side for me to reach it anytime I want. And If I don't want to see it when I am reading I can just close it.

Bonus: A hotkey to let me toggle the Side Panel open/close.

**Comment by BenjaminMichaelis** [comment, marked as answer]
Thanks for the feedback @brucegomes, this should be fixed!
